lib/v2/screens/sign_up/claim_invite/components/invite_link_fail_dialog.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/i18n/security.i18n.dart';
+import 'package:seeds/v2/components/custom_dialog.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+
+class InviteLinkFailDialog extends StatelessWidget {
```

Comment:
If we have more than 1 such error popup, it should be a common class with parameters, e.g. title and body text....

Reply:
in fact we have a custom dialog I'm not sure why I didn't use it here.